app/views/service_providers/show.html.erb

```diff
@@ -29,7 +29,7 @@
       <p class="font-mono-xs margin-top-0" name="production_issuer"><%=  service_provider.production_issuer %></p>
     <% end %>
 
-<% if Figaro.env.logo_upload_enabled == 'true' %>
+<% if ENV.fetch('logo_upload_enabled') { 'true' } == 'true' %>
```

Comment:
I'd just do the direct form not the block form: `ENV.fetch('logo_upload_enabled', 'true') == 'true')`

Reply:
ditto for the other spots too